app/data/session-data-defaults.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  account: {
+    email: 'janina.doe@example.com'
```

Comment:
I think we need to mark this as `disabled_account` too.

Reply:
Fixed and force-pushed.